bindings/matrix-sdk-ffi/src/api.udl

```diff
@@ -149,6 +149,16 @@ interface SlidingSyncBuilder {
     SlidingSync build();
 };
 
+dictionary CreateRoomParameters {
+    string name;
+    string? topic;
+    boolean is_direct;
+    Visibility visibility;
+    Preset preset;
```

Comment:
maybe is worth creating a wrapper ffi rust types for Visibility and Preset, that can be converted into ruma types

Reply:
@jplatte Mauro's proposal sounds good to you?

Reply:
If we only need to convert *to* the Ruma types, not *from* them, we can have enums in the FFI crate. Otherwise, I'd just use `String` which can be converted to and from the Ruma types.

Reply:
Applied -> https://github.com/matrix-org/matrix-rust-sdk/pull/1635/commits/7c51626d1b9343ace9e68895af15ffe6ca196895